dracut/image-boot/eos-image-boot-setup

```diff
@@ -113,7 +113,8 @@ iso9660)
   fi
 
   # Create a loopback device backing the squashfs
-  if ! image_device=$(losetup --find --show /cd/${image_path}); then
+  image_device=$(losetup -f)
+  if ! image_device=$(losetup ${image_device} /cd/${image_path}); then
```

Comment:
There's a race here of course – what if something else uses the loop device between the two invocations of `losetup`? In practice we don't expect anything else to be using loop devices at this point in the boot process so I guess it's OK.